scipy/stats/mstats_basic.py

```diff
@@ -622,7 +622,7 @@ def kendalltau(x, y, use_ties=True, use_missing=False, method='auto'):
             prob = 2.0/np.math.factorial(n-1)
         elif 2*c == (n*(n-1))//2:
             prob = 1.0
-        else:
+        elif n < 171:
```

Comment:
If you change the cutover to something much smaller, e.g. `n=1`, does the new code return the same values as the original `n<171` code?